TestFoundation/main.swift

```diff
@@ -23,6 +23,7 @@ internal func testBundle() -> NSBundle {
 XCTMain([
     TestNSString(),
     TestNSArray(),
+    TestNSCalendar(),
```

Comment:
TestNSCalendar.swift needs to be added to build.py in order to maintain building for linux as well.